README.md

```diff
@@ -269,7 +269,55 @@ Which is the short-hand equivalent to using `useStoryblokApi` inside `useState`
 
 > The `useState` is an SSR-friendly `ref` replacement. Its value will be preserved after server-side rendering (during client-side hydration).
 
-## Rendering Rich Text
+### Rendering Rich Text
+
+You can render rich-text fields by using the `StoryblokRichText` component:
```

Comment:
```suggestion
You can render rich text fields by using the `StoryblokRichText` component:
```

@alvarosabu I proposed to remove the dash, for consistency with how we refer to "rich text" along the readme